website/static/css/pikaday.css

```diff
@@ -0,0 +1,203 @@
+@charset "UTF-8";
+
+/*!
```

Comment:
After double checking, it does. How do we import css files that come from the node installation folder(s)? I didn't see it done anywhere else so I just manually copied it over with the rest.